script.js

```diff
@@ -185,4 +185,35 @@ feature4.addEventListener('click', () => {
 
 IconClose4.addEventListener('click', () => {
   popup4.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+/* Contact form */
+const ContactForm = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const nameF = document.getElementById('name');
+const message = document.getElementById('message');
+const errorMsg = document.getElementById('errormsg');
```

Comment:
- [ ] The `errorMsg` constant is null because there is not `id` named `errormsg` in your `index.html`. Kindly check it.
**HIT:** `const errorMsg = document.getElementById('errorMsg')`